### PR TITLE
[Notifier] Add Push Channel with Integration Expo & OneSignal

### PR DESCRIPTION
**Pull request by zairigimad**
Adding documentation for 

- https://github.com/symfony/symfony-docs/issues/15946
- https://github.com/symfony/symfony-docs/issues/16027

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Thanks a lot Imad!

File: notifier.rst [hunk 1]
```diff
@@ -36,6 +36,7 @@ The notifier component supports the following channels:
   services like Slack and Telegram;
 * :ref:`Email channel <notifier-email-channel>` integrates the :doc:`Symfony Mailer </mailer>`;
 * Browser channel uses :ref:`flash messages <flash-messages>`.
+* Push Channel sends notifications to phones via push notifications.
```
Comment (norkunas) [inline notifier.rst:39]: I'd say not necessary to phones as it's possible to subscribe push notifications also via browser..

Comment (norkunas) [reply notifier.rst:39]: also thanks for writing docs for this :)

Comment (javiereguiluz) [reply notifier.rst:39]: Fixed in https://github.com/symfony/symfony-docs/commit/263bdd7f821a1208c70f1b64a48c0957516595f0 Thanks!

**Pull request merged**